Table/models/table-column.model.ts

```diff
@@ -324,6 +324,7 @@ export class TableColumn<T = IItem> implements IItemBase<T> {
       case 'date':
       case 'timestamp':
       case 'duration':
+      case 'stringEnum':
```

Comment:
Shouldn't be part of the `components`

Reply:
Leaving after dm discussion - there is no good alternative present.